kernel/fde/fde_test.go

```diff
@@ -499,29 +382,24 @@ func (s *fdeSuite) TestRevealErr(c *C) {
 
 	mockSystemdRun := testutil.MockCommand(c, "systemd-run", `echo failed 1>&2; false`)
 	defer mockSystemdRun.Restore()
-	restore := fde.MockFdeInitramfsHelperCommandExtra([]string{"--user"})
-	defer restore()
 
 	sealedKey := []byte{1, 2, 3, 4}
 	p := fde.RevealParams{
 		SealedKey: sealedKey,
 	}
 	_, err := fde.Reveal(&p)
-	c.Assert(err, ErrorMatches, `(?s)cannot run fde-reveal-key "reveal": failed`)
+	c.Assert(err, ErrorMatches, `(?s)cannot run [[]"fde-reveal-key"[]]: failed`)
```

Comment:
(nitpick) Maybe just escape `[` as `\[` which is what I believe we do in most similar cases? Or is there a advantage of using the `[[]` version?

Reply:
I usually use `[[]` and `[]]` because I never have to worry about double escapes whatever the language I use, or whether it is a raw string or normal string. But I can change that.

Reply:
Done

Reply:
Thanks for explaining the advantages, it's good reasoning, I will keep it in mind!